### ISSUE TITLE
About looking for the original sentences of the triples

### ISSUE THREAD

**Issue by Frankie123421** [samuelbroscheit/open_knowledge_graph_embeddings#3]
Hey, I am recently studying your work. I would like to know if there exists some convenient approach to obtain the original sentences of the triples in the dataset. Must I look for them from OPIEC? Thank you.

**Comment by samuelbroscheit** [comment]
That is correct, you have to look at OPIEC. You can reuse the OLPBENCH pipeline (i.e. especially this part https://github.com/samuelbroscheit/open_knowledge_graph_embeddings/blob/main/preprocessing/process_avro.py ) and extend it to also create an index that associates each triple with its source sentence.

**Comment by Frankie123421** [comment]
Thanks for your kind reply, I got it. So maybe I can also just concatenate the "word" in the "tokens" to directly obtain the sentence, is that correct? I don't know whether this would be more convenient than using the "sentence number" to further look for the sentence in the article, suppose that I just want to find the corresponding original sentences of the triples. By the way, I feel that maybe it will also be difficult to match the triples in this dataset to OPIEC by simply using loops for OPIEC is quite large?

**Comment by EsratRupu** [comment]
Hey, I am trying to extract the source sentence of OLPBench triples. Therefore, I followed the procedure which is mentioned "Create OLPBENCH from scratch" section. But when I run the **create_data.py** I found that in the log the length of these two files is 0 (**triples_list_lc 0, triples_sro_to_se_oe_dict 0**). I also check the .pickle files. Those files are also empty. Would you please check this issue?
